pybossa/api/__init__.py

```diff
@@ -395,10 +395,11 @@ def task_progress(project_id=None, short_name=None):
     for key in filter_fields.keys():
         if key in task_fields:
             sql_text += " AND {0}=:{1}".format(key, key)
-        elif key in task_info_fields and filter_fields[key] == "null".lower():
-            sql_text +=  " AND info ->> '{0}' is Null".format(key)    
         elif key in task_info_fields:
-            sql_text += " AND info ->> '{0}'=:{1}".format(key, key)
+            if filter_fields[key].lower() == "null":
```

Comment:
Should the API support an empty string value?

`/api/project/1/taskprogress?Fruit=`

I'm thinking the value `null` is probably sufficient, just curious. Thanks.

Reply:
Currently, 
/api/project/1/taskprogress?Fruit=
is not supported. 
Null is currently the supported keyword for querying empty string + null